### PR TITLE
Optionally change the help view option alias separator

### PR DESCRIPTION
**Pull request by seancpeters**

### CODE REVIEW

File: CommandLine.Tests/HelpViewTests.cs [hunk 1]
```diff
@@ -303,5 +303,17 @@ public void When_unmatched_tokens_are_allowed_then_help_view_indicates_it()
...
+        [Fact]
+        public void When_option_alias_separator_is_provided_it_is_used()
+        {
+            var command = Command("command", "Help text",
+                Option("-l|--list", "Help for list option"),
+                Option("-a|--all", "Help for all option"));
+            HelpViewOptions helpViewOptions = new HelpViewOptions("| ");
+            var helpView = command.HelpView(helpViewOptions);
+            helpView.Should().Contain("-l| --list");
+            helpView.Should().Contain("-a| --all");
```
Comment (jonsequitur) [inline CommandLine.Tests/HelpViewTests.cs]: We may want to hold off on this. We've also changed the format in the dotnet CLI (though we're still discussing the final form) to use a comma in the help view, as well as some other tweaks to bring it more in line with the online documentation (https://github.com/dotnet/cli/issues/5902). The comma seems more common in other CLI tools.

Additionally, we have an alternate design in mind for how to do this kind of help view customization. Adding an option for every possible tweak won't get us very far. We should discuss.

@livarcocc

Comment (seancpeters) [inline CommandLine.Tests/HelpViewTests.cs]: Ok, sounds good. I'll be interested in seeing / hearing about the alternate design.

Comment (wli3) [inline CommandLine.Tests/HelpViewTests.cs]: @jonsequitur we should go with `,` now?

Comment (jonsequitur) [inline CommandLine.Tests/HelpViewTests.cs]: We agreed the other day to consolidate on the new behavior, using the comma.

**Pull request closed without merging**